CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ enable_testing()
 
 set(GTEST_VERSION 1.12.1)
 
-option(MENDER_DOWNLOAD_GTEST "Download google test if it is not found (Default: Yes)" ON)
+option(MENDER_DOWNLOAD_GTEST "Download google test if it is not found (Default: ON)" ON)
```

Comment:
Did I trip you up? :p

Reply:
This is because I remember an earlier open source project I built where this was driving me crazy. I tried `true`, `TRUE`, `1`, `on`, `yes`, `YES`, and probably several others before I finally realized what it should be. Let's give the next guy the soft way, instead of the hard way. 🙂